src/jit/gschecks.cpp

```diff
@@ -473,9 +473,9 @@ void Compiler::gsParamsToShadows()
             dst = gtNewOperNode(GT_ADDR, TYP_BYREF, dst);
 
             opAssign = gtNewCpObjNode(dst, src, clsHnd, false);
-#if FEATURE_MULTIREG_STRUCTS
-            lvaTable[shadowVar].lvDontPromote = lvaTable[lclNum].lvDontPromote;
-#endif // FEATURE_MULTIREG_STRUCTS
+#if FEATURE_MULTIREG_ARGS_OR_RET
+            lvaTable[shadowVar].lvIsMultiRegArgOrRet = lvaTable[lclNum].lvIsMultiRegArgOrRet;
```

Comment:
Would it still make sense to copy lvDontPromote field from original arg to shadow arg outside #ifdef in general?

If for whatever reasons we marked an arg as don't promote, we want the same to be enforced on its corresponding shadow arg.